Source/SvgDocument.cs

```diff
@@ -260,32 +260,35 @@ private static bool ExceptionCaughtIsGdiPlusRelated(Exception e)
         /// Opens the document at the specified path and loads the SVG contents.
         /// </summary>
         /// <param name="path">A <see cref="string"/> containing the path of the file to open.</param>
+        /// <param name="baseUri"> base uri for linked file</param>
         /// <returns>An <see cref="SvgDocument"/> with the contents loaded.</returns>
         /// <exception cref="FileNotFoundException">The document at the specified <paramref name="path"/> cannot be found.</exception>
-        public static SvgDocument Open(string path)
+        public static SvgDocument Open(string path, Uri baseUri = null)

```

Comment:
Adding `Open` argument doesn't seem to be active. (ref. #457)

Reply:
i don't understand.